integration-tests/go.mod

```diff
@@ -137,3 +139,7 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 // Fix go mod tidy issue for ambiguous imports from go-ethereum
 // See https://github.com/ugorji/go/issues/279
 replace github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.1
+
+//replace github.com/smartcontractkit/chainlink-starknet/ops => ../ops
+
+//replace github.com/smartcontractkit/chainlink-env => ../../chainlink-env
```

Comment:
There is a `make gowork` command that would initialize your workspace and you can add more replaces there, helpful so you don't pollute your PRs as `go.work` is also in `.gitignore`.